src/utils/record-utils.ts

```diff
@@ -360,3 +361,24 @@ export function getPrefillCookieObject(ref: any, mainTuple: any): {
     keys: keys
   };
 }
+
+export function determineScrollElement(displayname: string): Element | false {
```

Comment:
Please add a comment for this function and summarize its purpose. Its name is very general but it seems like it's specifically implemented for finding the section/related column element for scrolling.